packages/wp-story-editor/src/components/statusCheck/statusCheck.js

```diff
@@ -39,10 +39,22 @@ function StatusCheck() {
   useEffect(() => {
     // If it succeeds, do nothing.
     // Only in case of failure do we want to alert the user and track the error.
-    getStatusCheck(getContent(), statusCheck, encodeMarkup).catch((err) => {
-      setShowDialog(true);
-      trackError('status_check', err.message);
-    });
+    getStatusCheck(getContent(), statusCheck, encodeMarkup)
```

Comment:
Would making this async improve readability?

Reply:
I don't think so, but open to suggestions if you see a benefit.